src/odemis/util/__init__.py

```diff
@@ -24,28 +24,43 @@
 # Various helper functions that have a generic usefulness
 # Warning: do not put anything that has dependencies on non default python modules
 
-from __future__ import division, absolute_import
+from __future__ import absolute_import, division
 
-import queue
 import collections
-from concurrent.futures import CancelledError
-from decorator import decorator
-from functools import wraps
 import inspect
+import itertools
 import logging
 import math
-import numpy
+import queue
 import signal
 import sys
 import threading
 import time
-import weakref
 import types
+import weakref
+from concurrent.futures import CancelledError
+from functools import wraps
+from typing import Iterable, Tuple, TypeVar
+
+import numpy
+from decorator import decorator
```

Comment:
Ideally, minimize the changes to the import part which is not affecting any behaviour. It's much less clear which import you have added/removed.

Reply:
The code got passed through `isort` as a pre-commit. Do you want me to revert to the original ordering?

Reply:
Yes please revert. Now that there should be very little changes, I think it's much easier to read your changes without the resorting.
I don't recommend having some sort of automatic code rewriting before your commit, as it's hard to control. I'd recommend having it part of your IDE.